src/server/enrollment-system.js

```diff
@@ -362,6 +359,10 @@ function veteranToDependentFinancialsCollection(veteran) {
  * @returns {Object} ES system dependentFinancialsCollection message
  */
 function veteranToSpouseFinancials(veteran) {
+  if (!_.includes(['Married', 'Separated'], veteran.maritalStatus)) {
```

Comment:
Gotcha. Okay now I'm seeing that `dependentFinancials` and `numberOfDependentChildren` are being included elsewhere. Was just a little confused to see those returned with spouseFinancials.